setup.py

```diff
@@ -396,6 +396,7 @@ def run(self):
             'azure-functions==1.7.2',
             'azure-eventhub~=5.1.0',
             'python-dateutil~=2.8.1',
+            'pycryptodome~=3.10.1',
```

Comment:
This library is used for generating **encryptedContext** when specializing a placeholder in Linux Consumption container. See `_encrypt_context` implementation.

Reply:
License BSD: https://github.com/Legrandin/pycryptodome/blob/master/LICENSE.rst#bsd-license